mods/cnc/rules/infantry.yaml

```diff
@@ -211,10 +211,13 @@ E6:
 	Captures@SABOTAGE:
 		CaptureTypes: building-sabotage
 		SabotageThreshold: 55
-		PlayerExperience: 50
+		PlayerExperience: 5
```

Comment:
```suggestion
		PlayerExperiencePercentage: 5
```
?

Reply:
Oh my bad. I though sabotage was only the damage, and the capturing part was below